iothub_client/src/iothub_client_ll.c

```diff
@@ -461,8 +466,8 @@ static IOTHUB_DEVICE_TWIN* dev_twin_data_create(IOTHUB_CLIENT_LL_HANDLE_DATA* ha
             result->ms_timesOutAfter = 0;
             result->context = userContextCallback;
             result->reported_state_callback = reportedStateCallback;
-			result->client_handle = handleData;
-			result->device_handle = handleData->deviceHandle;
+            result->client_handle = handleData;
+            result->device_handle = handleData->deviceHandle;
```

Comment:
Why these changes are included? please revert.

Reply:
The change must be introduced by formatting the file.

It was updated with **correct** indent spaces, do you want me to revert it and use separate PR to commit the change?